<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCubeRTC</class>
 <widget class="QWidget" name="GameCubeRTC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameCube RTC</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelStartSeed">
       <property name="text">
        <string>Origin Seed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="TextBox" name="textBoxStartSeed"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelMinAdvance">
       <property name="text">
        <string>Min Advance</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="TextBox" name="textBoxMinAdvance">
       <property name="text">
        <string notr="true">500000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEndSeed">
       <property name="text">
        <string>Target Seed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="TextBox" name="textBoxTargetSeed"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelMaxAdvance">
       <property name="text">
        <string>Max Advance</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="TextBox" name="textBoxMaxAdvance">
       <property name="text">
        <string notr="true">1000000</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelEndDate">
       <property name="text">
        <string>End Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="DateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="TableView" name="tableView">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>Forms/Controls/DateEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textBoxStartSeed</tabstop>
  <tabstop>textBoxTargetSeed</tabstop>
  <tabstop>textBoxMinAdvance</tabstop>
  <tabstop>textBoxMaxAdvance</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
